kombu/transport/SQS.py

```diff
@@ -104,6 +104,10 @@ def __init__(self, *args, **kwargs):
         self.hub = kwargs.get('hub') or get_event_loop()
 
     def _update_queue_cache(self, queue_name_prefix):
+        if self.predefined_queue_urls:
```

Comment:
You will possibly need one small test to verify the change in the behavior of `_update_queue_cache` depending on the transport settings. Let me know if you need help with that!

Reply:
I probably will need some help with that - I haven't contributed to Python projects in a while, so not up to speed on the latest testing...

Reply:
The unit tests for the SQS transport can be found [here](https://github.com/celery/kombu/blob/master/t/unit/transport/test_SQS.py). I will try to construct a small test case.